arithmetization/src/main/java/net/consensys/linea/zktracer/module/hub/section/halt/SelfdestructSection.java

```diff
@@ -84,15 +84,10 @@ public SelfdestructSection(Hub hub) {
     selfdestructorAccountBefore = AccountSnapshot.canonical(hub, addressWhichMaySelfDestruct);
 
     // Recipient
-    recipientRawAddress = frame.getStackItem(0);
-    recipientAddress = Address.extract(Bytes32.leftPad(recipientRawAddress));
+    recipientAddressUntrimmed = frame.getStackItem(0);
+    recipientAddress = Address.extract(Bytes32.leftPad(recipientAddressUntrimmed));
```

Comment:
stupid question, don't we need to trigger TRM ? Id on't see it triggered in the section

Reply:
No, you are right. We trigger it in the OOGX case (`makeWithTrm`) but not in the unexceptional case (`make`). It's fixed now.